extensions/src/main/java/org/pentaho/platform/web/http/api/resources/JobScheduleRequest.java

```diff
@@ -101,6 +101,10 @@ public class JobScheduleRequest implements Serializable {
 
   String timeZone;
 
+  protected String runSafeMode;
+  protected String gatheringMetrics;
```

Comment:
@srallapa You have some checkstyle issues. Please run our checkstyle in Intellij for all .java files using these instructions:
the README.md file in https://github.com/pentaho/pentaho-coding-standards

Reply:
Did changes based on checkstyle